### DISCUSSION TITLE
Is it possible to detonate a nuclear charge (atomic bomb or railgun shell) using a detonator?

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13690] [Ideas and suggestions]
For example, if you detonate a UEX, C-4 or compound N next to (an atomic bomb or a railgun shell), will a nuclear explosion occur?
Real atomic bombs do not detonate. If you blow them up, you'll be left with a pile of radioactive uranium dust.

**Comment by SomeRandomNoobKekeke** [comment]
https://www.youtube.com/watch?v=Wmvycx3MSOM

**Comment by Regalis11** [comment]
> Real atomic bombs do not detonate. If you blow them up, you'll be left with a pile of radioactive uranium dust.

Real atomic bombs aren't launched from railguns nor set to trigger when they hit something or when a monster touches them. To me it seems logical that the explosion would cause a high enough impulse to trigger the nuclear shell/charge. And it works from a gameplay perspective, being able to trigger nuclear explosives with conventional explosives opens up possibilities for creative use of nukes.

**Comment by pavel4win** [comment]
So nukes can be detonated with other explosives. It is sounds good for me. Thank for quick reply.
